projects/1.18/assets/decoration-delight/decoration_delight/lang/zh_cn.json

```
@@ -0,0 +1,79 @@
+  "block.decoration_delight.birch_counter": "桦木柜台",
+  "block.decoration_delight.birch_counter_bend": "桦木柜台转角",
+  "block.decoration_delight.birch_counter_corner": "桦木柜台角落",
+  "block.decoration_delight.birch_countertop": "桦木台面",
+  "block.decoration_delight.birch_mosaic": "桦木马赛克",
+  "block.decoration_delight.birch_stool": "桦木凳",
+  "block.decoration_delight.crimson_counter": "绯红木柜台",
+  "block.decoration_delight.crimson_counter_bend": "绯红木柜台转角",
+  "block.decoration_delight.crimson_counter_corner": "绯红木柜台角落",
+  "block.decoration_delight.crimson_countertop": "绯红木台面",
+  "block.decoration_delight.crimson_mosaic": "绯红木马赛克",
+  "block.decoration_delight.crimson_stool": "绯红木凳",
+  "block.decoration_delight.dark_oak_counter": "深色橡木柜台",
+  "block.decoration_delight.dark_oak_counter_bend": "深色橡木柜台转角",
+  "block.decoration_delight.dark_oak_counter_corner": "深色橡木柜台角落",
+  "block.decoration_delight.dark_oak_countertop": "深色橡木台面",
+  "block.decoration_delight.dark_oak_mosaic": "深色橡木马赛克",
+  "block.decoration_delight.dark_oak_stool": "深色橡木凳",
+  "block.decoration_delight.diorite_tiles": "闪长岩瓷砖",
+  "block.decoration_delight.granite_tiles": "花岗岩瓷砖",
+  "block.decoration_delight.jungle_counter": "丛林木柜台",
+  "block.decoration_delight.jungle_counter_bend": "丛林木柜台转角",
+  "block.decoration_delight.jungle_counter_corner": "丛林木柜台角落",
+  "block.decoration_delight.jungle_countertop": "丛林木台面",
+  "block.decoration_delight.jungle_mosaic": "丛林木马赛克",
+  "block.decoration_delight.jungle_stool": "丛林木凳",
+  "block.decoration_delight.oak_counter": "橡木柜台",
+  "block.decoration_delight.oak_counter_bend": "橡木柜台转角",
+  "block.decoration_delight.oak_counter_corner": "橡木柜台角落",
+  "block.decoration_delight.oak_countertop": "橡木台面",
+  "block.decoration_delight.oak_mosaic": "橡木马赛克",
+  "block.decoration_delight.oak_stool": "橡木凳",
+  "block.decoration_delight.quartz_and_blackstone_tiles": "黑白瓷砖",
+  "block.decoration_delight.sink": "洗碗槽",
+  "block.decoration_delight.spice_collection": "香料瓶",
+  "block.decoration_delight.spruce_counter": "云杉木柜台",
+  "block.decoration_delight.spruce_counter_bend": "云杉木柜台转角",
+  "block.decoration_delight.spruce_counter_corner": "云杉木柜台角落",
+  "block.decoration_delight.spruce_countertop": "云杉木台面",
+  "block.decoration_delight.spruce_mosaic": "云杉木马赛克",
+  "block.decoration_delight.spruce_stool": "云杉木凳",
+  "block.decoration_delight.stone_tiles": "石头瓷砖",
+  "block.decoration_delight.trash_bin": "垃圾桶",
+  "block.decoration_delight.warped_counter": "诡异木柜台",
+  "block.decoration_delight.warped_counter_bend": "诡异木柜台转角",
+  "block.decoration_delight.warped_counter_corner": "诡异木柜台角落",
+  "block.decoration_delight.warped_countertop": "诡异木台面",
+  "block.decoration_delight.warped_mosaic": "诡异马赛克",
+  "block.decoration_delight.warped_stool": "诡异木凳",
+  "entity.decoration_delight.stool_entity": "凳子",
+  "gui.decoration_delight.cabinet_storage.label_cabinet": "储藏柜",
+  "gui.decoration_delight.cabinet_storage.label_inventory": "库存",
```

`Inventory`在这里怎么处理好一点
```suggestion
  "gui.decoration_delight.cabinet_storage.label_inventory": "物品栏",
```

---

游戏内用于gui的玩家背包一栏，确实是物品栏合适